buildkite/bazelci.py

```diff
@@ -720,6 +724,20 @@ def _open_url(self, url, params=[]):
         except urllib.error.HTTPError as ex:
             raise BuildkiteException("Failed to open {}: {} - {}".format(url, ex.code, ex.reason))
 
+    def get_pipeline_info(self):
+    """Get details for a pipeline given its organization slug
```

Comment:
Oh, we need to fix the indentation here.

Reply:
Fixed, sorry about that.